apps/src/applab/dropletConfig.js

```diff
@@ -17,7 +17,9 @@ var DEFAULT_HEIGHT = (480 - applabConstants.FOOTER_HEIGHT).toString();
 
 // Flip the argument order so we can bind `typeFilter`.
 function chooseAsset(typeFilter, callback) {
-  dashboard.assets.showAssetManager(callback, typeFilter, null, !studioApp.reduxStore.getState().pageConstants.is13Plus);
+  dashboard.assets.showAssetManager(callback, typeFilter, null, {
+    showUnderageWarning: !studioApp.reduxStore.getState().pageConstants.is13Plus
+  });
```

Comment:
In fact, this happens identically a couple of times.  Any reason we couldn't/shouldn't extract this call (with the `showUnderageWarning` option) into a helper?